Domains/QuantityTakeoffsAspects/QuantityTakeoffsAspects.ecschema.xml

```diff
@@ -186,6 +212,11 @@
     
     <ECRelationshipClass typeName="ElementOwnsPipeAspect" strength="embedding" modifier="None" description="A relationship tieing the specialized quantity takeoff aspect with an element">
         <BaseClass>ElementOwnsQuantityTakeoffAspect</BaseClass>
+        <ECCustomAttributes>
+            <Deprecated xmlns="CoreCustomAttributes.01.00.03">
+                <Description>Use Generic:PhysicalObject instead with a descriptive Generic:PhysicalType.</Description>
```

Comment:
description is not clear? what's a descriptive physicaltype? 

how do I build a query to select pipes?

(currently select bis.physicalobject join pipeaspect)

Reply:
Scaled back this deprecation to only classes that have full equivalences on other schemas.